format/proto/types.proto

```diff
@@ -28,13 +28,35 @@ message Metadata {
     optional sfixed32 length = 2;
 }
 
-//  NodeMetrics is composed of the various metrics of a node
+//  ClusterView is a node's view of the cluster.
+//  This is sent in response to a heartbeat request by a
+//  Management Server.
+//  It stores its own endpoint and server metrics and also
+//  the network condition of all its peers.
+message ClusterView {
```

Comment:
Changed this.

**NodeView** is the new object containing the node's serverMetrics (component ping status, write queue and token count) and the node's connectivity to its peers (networkMetrics map)

We have a HeartbeatPollReport which is returned by the polling service.